cmd/skupper/skupper.go

```diff
@@ -440,10 +441,22 @@ installation that can then be connected to other skupper installations`,
 				return fmt.Errorf("The --enable-flow-collector option must be used with the --enable-console option")
 			}
 
+			if routerCreateOpts.AuthMode != "internal" && (len(routerCreateOpts.User) > 0 || len(routerCreateOpts.Password) > 0) {
```

Comment:
The default mode is internal, although the default value is empty. So with this
change it is forcing the --console-auth to be provided. I believe instead of checking
if the value is not "internal" it should also consider AuthMode as not being empty,
or eventually you could set a default value to `routerCreateOpts.AuthMode`.

Reply:
1. There is no explicit default value, but if --console-auth is empty, the default is `internal`. 
2. The documentation states that --console-auth is mandatory (#1149). That's why I was forcing the value to be provided.

Probably the best would be to:
- Make an explicit default value aligned with the code
- Eliminate 51473eeb635042d0c3aba392f28729c52875aed2 otherwise it could confuse  users that could be not interested in the console/flow-collector
- Update the documentation accordingly

what do you think?

Reply:
It makes sense to me!

Reply:
Updated in 48a5d32b299a44b2bffb011999c0d6664e2bc1fe